terminal.go

```diff
@@ -1112,6 +1129,18 @@ func (g *LightningTerminal) shutdownSubServers() error {
 		}
 	}
 
+	if g.rpcProxy != nil {
+		if err := g.rpcProxy.Stop(); err != nil {
+			log.Errorf("Error stopping rpc proxy: %v", err)
+		}
+	}
+
+	if g.httpServer != nil {
+		if err := g.httpServer.Close(); err != nil {
+			log.Errorf("Error stopping UI server: %v", err)
+		}
+	}
```

Comment:
I think we should also set the `returnErr` here if `g.rpcProxy.Stop()` or `g.httpServer.Close()` fails?